### PR TITLE
XWIKI-15825: Use feature and grouping parameters to the Include macro descriptor

### PR DESCRIPTION
**Pull request by atallahade**
### Issue

https://jira.xwiki.org/browse/XWIKI-15825

### Changes

* Apply the PropertyGroup and PropertyFeature to some include macro
  parameters.

### CODE REVIEW

File: xwiki-platform-core/xwiki-platform-rendering/xwiki-platform-rendering-macros/xwiki-platform-rendering-macro-include/src/main/java/org/xwiki/rendering/macro/include/IncludeMacroParameters.java [hunk 1]
```diff
@@ -72,6 +74,8 @@
      * @since 3.4M1
      */
     @PropertyDescription("the reference of the resource to display")
+    @PropertyGroup({"stringReference"})
```
Comment (tmortagne) [inline xwiki-platform-core/xwiki-platform-rendering/xwiki-platform-rendering-macros/xwiki-platform-rendering-macro-include/src/main/java/org/xwiki/rendering/macro/include/IncludeMacroParameters.java]: You don't need the {} when there is only one.

Comment (atallahade) [reply xwiki-platform-core/xwiki-platform-rendering/xwiki-platform-rendering-macros/xwiki-platform-rendering-macro-include/src/main/java/org/xwiki/rendering/macro/include/IncludeMacroParameters.java]: Good to know, I thought this feature had to be implemented in the annotation itself.

**Comment by evalica** [comment]
This issue is supposed to be fixed in the next release. Can someone else approve the changes? Thanks.

**Comment by atallahade** [comment]
I can merge it without any problem I think. I just wanted to show @vmassol and @mflorea if this was actually what we wanted to do.

**Pull request merged**